training/distributed_training/pytorch/data_parallel/rnnt/RNNT_notebook.ipynb

```diff
@@ -0,0 +1,371 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    region = "us-west-2"  # Example: us-west-2
</pre><p>Set region to be whatever the current default region is for the customer.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
fixed!